dashboard/test/controllers/api/v1/schools_controller_test.rb

```diff
@@ -29,33 +29,21 @@ class Api::V1::SchoolsControllerTest < ActionController::TestCase
     assert_equal [GLADYS_JUNG_ELEMENTARY], JSON.parse(@response.body)
   end
 
-  test 'search by school name suffix' do
-    get :search, params: {q: 'tary', limit: 40}
-    assert_response :success
-    assert_equal [ALBERT_EINSTEIN_ACADEMY_ELEMENTARY, GLADYS_JUNG_ELEMENTARY], JSON.parse(@response.body)
-  end
-
   test 'search by school city prefix' do
     get :search, params: {q: 'beth', limit: 40}
     assert_response :success
     assert_equal [GLADYS_JUNG_ELEMENTARY], JSON.parse(@response.body)
   end
 
-  test 'search by school zip' do
-    get :search, params: {q: '99559', limit: 40}
+  test 'search with limit of negative one' do
+    get :search, params: {q: 'glad', limit: -1}
```

Comment:
passing `LIMIT -1` to mysql query seems to be causing a query-error test failure

Reply:
I had fixed that in a later push, -1 and 0 should default to minimum of 1.